analysis_options.yaml

```diff
@@ -1,25 +1,38 @@
-include: package:lints/recommended.yaml
+# https://dart.dev/guides/language/analysis-options
+include: package:dart_flutter_team_lints/analysis_options.yaml
+
+analyzer:
+  language:
+    strict-casts: true
+  errors:
+    # For the generated file
+    lines_longer_than_80_chars: ignore
```

Comment:
Ideally generated files would ignore this, so we don't have to opt out the entire repo? Or possibly the lint could be updated to recognize generated files? Just wondering if we should file a bug here really, this is fine for now.

Reply:
Sadly, the newly generated content fails even MORE lints.

But we also want the generated file to have some checks.

Reply:
fwiw, you can opt generated files out of many checks via `// ignore_for_file: type=lint`.

Reply:
oh nice!

Reply:
We'd need to teach the proto compiler this.

Reply:
yeah it would be nice to do though, I filed https://github.com/google/protobuf.dart/issues/803